include/prereq-build.mk

```diff
@@ -142,9 +142,15 @@ $(eval $(call SetupHostCommand,perl,Please install Perl 5.x, \
 	perl --version | grep "perl.*v5"))
 
 $(eval $(call SetupHostCommand,python,Please install Python 2.x, \
-	python2.7 -V 2>&1 | grep Python, \
-	python2 -V 2>&1 | grep Python, \
-	python -V 2>&1 | grep Python))
+	python2.7 -V 2>&1 | grep 'Python 2.7', \
+	python2 -V 2>&1 | grep 'Python 2', \
+	python -V 2>&1 | grep 'Python 2'))
+
+$(eval $(call SetupHostCommand,python3,Please install Python >= 3.5, \
```

Comment:
This should go into separate commit, it's not clear why do you need this for 19.07 which is python2 only.

Reply:
I want to put python3 as a requirement so that people can use it in their packages updates, but yes separate commit with an explanation is needed

Reply:
> I want to put python3 as a requirement

Don't waste time on that, it has zero chances to be accepted.

It has all started, because you had no python2 by default installed in your Fedora31, now you're going to add python3 requirement for everybody even if it's not needed by anything in the tree? If it's needed by some package in the feed, that please handle it there, at the proper place.